src/OidcClientSettings.js

```diff
@@ -20,6 +21,7 @@ export class OidcClientSettings {
         // client related
         client_id, client_secret, response_type = DefaultResponseType, scope = DefaultScope,
         redirect_uri, post_logout_redirect_uri,
+        client_authentication = DefaultClientAuthentication,
```

Comment:
The client_authentication is not defined in the dts file so the typescript is not recognising that client-authentication is a property of OidcClientSettings